modules/monitoring-example-remote-write-authentication-settings.adoc

```diff
@@ -79,7 +89,8 @@ data:
 <5> The name of the AWS profile that is being used to authenticate.
 <6> The unique identifier for the Amazon Resource Name (ARN) assigned to your role.
 
-[id="remote-write-sample-yaml-basic-auth_{context}"]
+//commenting this ID out for the build to not break. Will be uncommented after the migration.
+//[id="remote-write-sample-yaml-basic-auth_{context}"]
```

Comment:
Same comment about not commenting out IDs, especially in a module that's reused in the same assembly, even if this isn't a live branch. Please apply this feedback to all instances of commented out IDs